app/views/questionnaire.py

```diff
@@ -216,18 +211,20 @@ def get_timeout_continue(eq_id, form_type, collection_id):  # pylint: disable=un
     return 'true'
 
 
-@questionnaire_blueprint.route('session-expired', methods=["GET"])
+@questionnaire_blueprint.route('session-expired', methods=["POST"])
```

Comment:
The sign-out link on GitHub.com is a POST to `/logout`; which also raises the question of why ours is not on the root URL `/`, since our `sign-in` is i.e. `/session?token=....`